x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/EsqlFunctionRegistry.java

```diff
@@ -138,7 +142,10 @@ private FunctionDefinition[][] functions() {
                 def(DateTrunc.class, DateTrunc::new, "date_trunc"),
                 def(Now.class, Now::new, "now") },
             // conditional
-            new FunctionDefinition[] { def(Case.class, Case::new, "case"), def(IsNull.class, IsNull::new, "is_null"), },
+            new FunctionDefinition[] {
+                def(Case.class, Case::new, "case"),
+                def(Intersects.class, Intersects::new, "intersects"),
```

Comment:
I'd probably stick this in a `spatial` section. We have other predicates - like `is_nan` and stuff. Honestly I'm not sure what section `case` and `is_null` belong in - they are so funny.